packages/common/auth/src/services/CrossmintAuthService.ts

```diff
@@ -0,0 +1,25 @@
+import { APIErrorService, BaseCrossmintService } from "@crossmint/client-sdk-base";
+
+import { authLogger } from "./logger";
+
+export class CrossmintAuthService extends BaseCrossmintService {
```

Comment:
should be using `CrossmintApiClient`

Reply:
Oh okay, I'm going to refactor all of this anyway in another PR soon, so will leave as is for now